### ISSUE TITLE
Spiff Radio accessibility screen reader issues

### ISSUE THREAD

**Issue by Orinks** [gordielachance/spiff-radio-issues#1]
With a screen reader, I cannot access certain lists currently, such as editing playlist details, exporting them, etc. I also cannot create an account due to this issue.
Any help/fixes would be appreciated.
